fvm/evm/emulator/signer.go

```diff
@@ -0,0 +1,21 @@
+package emulator
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+var DefaultBlockNumberForEVMRules = big.NewInt(1) // anything bigger than 0
+
+// GetDefaultSigner returns a signer which is compatible with the default config
+func GetDefaultSigner() types.Signer {
+	cfg := NewConfig(WithBlockNumber(DefaultBlockNumberForEVMRules))
+	return GetSigner(cfg)
+}
+
+// GetSigner returns a signer that is compatible with the given config
+func GetSigner(cfg *Config) types.Signer {
```

Comment:
The name signer is confusing, it might be wise to rename to something like `SignerValidator` or at least explain in the comment. It was probably a bad naming in ethereum codebase like gas was :D

Reply:
I prefer to keep the same naming to be compatible with EVM but added some comments about naming at [c848b92](https://github.com/onflow/flow-go/pull/4927/commits/c848b9212ca242dd68b095c4a3cd63058af8c377)